### PR TITLE
Node.js code snippets for Rackspace CDN getting started guide

### PR DESCRIPTION
**Pull request by ycombinator**
Attn: @rdodev, @kenperkins

### CODE REVIEW

File: src/docs/cdn/getting-started/samples/create_service.rst [hunk 1]
```diff
@@ -12,7 +12,26 @@
 
 .. code-block:: javascript
 
-  // Not currently supported by this SDK
+  client.createService({
```
Comment (rdodev) [inline src/docs/cdn/getting-started/samples/create_service.rst]: this client init seems a bit too coupled. maybe create the object outside and pass it in, for the sake of clarity?
```
var opts = {
  name: 'name',
  domains: [{}],
  origins: [{}], ...
}

client.createService(opts, function (err) { ... });
```

**Comment by rdodev** [comment]
Other than feedback above, LGTM :sparkles:

File: src/docs/cdn/getting-started/samples/create_service.rst (hunk 1 above)
Comment (ycombinator) [inline src/docs/cdn/getting-started/samples/create_service.rst]: Good call. Changing.

File: src/docs/cdn/getting-started/samples/authentication.rst [hunk 2]
```diff
@@ -12,7 +12,13 @@
 
 .. code-block:: javascript
 
-  // Not currently supported by this SDK
+  pkgcloud = require('pkgcloud');
+
+  var rackspace = pkgcloud.database.createClient({
```
Comment (kenperkins) [inline src/docs/cdn/getting-started/samples/authentication.rst]: pretty sure it's not database

Comment (ycombinator) [inline src/docs/cdn/getting-started/samples/authentication.rst]: Hahaha, I don't know. That's debatable.

(Fixing)

**Comment by kenperkins** [comment]
lgtm :+1:

**Comment by ycombinator** [comment]
@kenperkins Fixed the `database` copy/paste fail. Ready for review again, please.

**Comment by kenperkins** [comment]
too slow

**Comment by ycombinator** [comment]
> too slow

what are you talking about? merging now :)

**Comment by kenperkins** [comment]
image

**Comment by ycombinator** [comment]
Hahaha, you know I'd do that.

**Pull request merged**